nototools/noto_fonts.py

```diff
@@ -272,6 +272,10 @@ def script_key_to_scripts(script_key):
     return frozenset(['Arab'])
   elif script_key == 'HST':
     raise ValueError('!do not know scripts for HST script key')
+  elif script_key == 'MONO':
+    # Mono doesn't actually support all of Latn, we need a better way to
+    # deal with pseud-script codes like this one.
```

Comment:
OK.  This is actually a major TODO that I've made attempts at a few times.  It's not easy getting the semantics right, and these codes are used all over the place.